src/encoded/tests/test_permissions.py

```diff
@@ -368,8 +368,9 @@ def test_submitter_post_update_experiment(submitter_testapp, lab, award, human_b
 def test_submitter_cant_post_other_lab(submitter_testapp, other_lab, award, exp_types):
     experiment = {'lab': other_lab['@id'], 'award': award['@id'], 'experiment_type': exp_types['microc']['@id']}
     res = submitter_testapp.post_json('/experiments-hi-c', experiment, status=422)
-    assert res.json['errors'][0]['name'] == 'Schema: lab'
-    assert "not in user submits_for" in res.json['errors'][0]['description']
+    # jsonschema - error ordering change
+    assert res.json['errors'][1]['name'] == 'Schema: lab'
+    assert "not in user submits_for" in res.json['errors'][1]['description']
```

Comment:
Maybe something like this (untested):
```suggestion
    assert any(error['name'] == 'Schema: lab' and "not in user submits_for" in error['description']
               for error in res.json['errors'])
```